tony-core/src/main/java/com/linkedin/tony/Constants.java

```diff
@@ -13,6 +13,11 @@
   // File Permission
   public static final FsPermission perm770 = new FsPermission((short) 0770);
 
+  // History Server related constants
+  public static final String JH_HOST= "ltx1-mlearn-alpha-jh01.grid.linkedin.com";
```

Comment:
don't put this....

Reply:
Hmm ok yeah I was thinking that we have our own private network so they can't even access it from outside, but it's good practice anw. I'll update this and other instances that uses real hostname/path/etc.